extensions/scarb-test-collector/src/main.rs

```diff
@@ -0,0 +1,45 @@
+use anyhow::Result;
+use camino::Utf8PathBuf;
+use clap::Parser;
+use std::str::FromStr;
+
+use scarb_metadata::MetadataCommand;
+
+use crate::compilation::compile_tests;
+use crate::crate_collection::collect_test_compilation_targets;
+use crate::metadata::{
+    corelib_for_package, dependencies_for_package, id_for_package, paths_for_package,
+};
+
+mod compilation;
+mod crate_collection;
+mod metadata;
+
+#[derive(Parser)]
+#[command(version)]
+struct Args {
+    #[arg(long)]
+    output_path: String,
+}
+
+fn main() -> Result<()> {
+    let args = Args::parse();
+    let scarb_metadata = MetadataCommand::new().inherit_stderr().exec()?;
+
+    let manifest_path = Utf8PathBuf::from_str(&std::env::var("SCARB_MANIFEST_PATH")?)?;
```

Comment:
Wouldn't it make more sense for you  to use `SCARB_PACKAGE` env, handled by scarb-ui's PackagesFilter?

Reply:
Is it passable to scarb when calling subcommand? Or should I just call pass it directly?